app/views/users/show.html.slim

```diff
@@ -126,9 +126,9 @@
                         p.my-1
                           em = "#{@user.name} heeft nog niet ingelegd en er zijn nog geen correcties"
                   - else
-                    - @user.credit_mutations.each do |mutation|
+                    - @user.credit_mutations.order(created_at: :desc).each do |mutation|
                       tr
                         th scope='row' = mutation.id
-                        td = l mutation.created_at
+                        td = l(mutation.created_at, format: "%d-%m-%Y %H:%M")
```

Comment:
Might be better to include a formatter instead of an explicit format. Then the explicit format can be added somewhere (in the locals is I am right). This is already done (iirc), so could you check that?

Reply:
It might even be the default `:long` format

Reply:
@cmitz Nope, that writes out everthing (for example '1 oktober 2019')